app/src/main/java/com/example/voyagerx/MainActivity.kt

```diff
@@ -39,5 +39,7 @@ class MainActivity : AppCompatActivity() {
             replace(R.id.frame, desiredFragment)
             commit()
         }
+
+        setContentView(R.layout.activity_main)
```

Comment:
I think this line is preventing the view from updating when the bottom nav buttons are pressed.

Reply:
@VivianLi98 why do we need to call set content here again?

@brownyte , does it work when commenting out this line?

Reply:
I was able to get the bottom navigation buttons to start working again once I removed that line in the branch I have been working in